Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell_Tests.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerCommandCellView_Tests: XCTestCase {
+    func test_defaultAppearance() {
+        let view = ChatMessageComposerCommandCellView()
+        view.command = .init(name: "Hello", description: "what", set: "is this", args: "WHAT")
+        AssertSnapshot(view, size: CGSize(width: 300, height: 40), record: true)
```

Comment:
It's important for cells to test they work correctly with dynamic sizing, so IMO it makes more sense to just add one fixed constraint (typically a width constraint) and then let the other dimension dynamic.

Reply:
In this case, the Suggestions View Controller, which I'm testing right now too, the command cell's have actually a fixed size 🤔 In this case in particular, I think the idea of the design is that if the command is too long, it should just truncate the text. It would look kinda weird that the size of the cell, in this case, would grow, that is why I ignored the self-sizing, WDYT ?

Reply:
@VojtaStavik Fixed ✅